.expeditor/scripts/finish_release/bump_rustfmt.sh

```diff
@@ -73,16 +73,26 @@ run_fmt_open_pr() {
   echo "--- :rust: Running new rustfmt"
   cargo +"$(< RUSTFMT_VERSION)" fmt --all
 
+  echo "--- :github: Committing Changes"
+
+  # We'll be Expeditor for this commit (which is who we would be if we
+  # had access to Expeditor's open_pull_request helper here).
+  git config user.name "Chef Expeditor"
+  git config user.email "expeditor@chef.io"
+
   git add --update
-  maybe_run git commit --signoff --message "\"Bump nightly toolchain to ${FOUND_VERSION}\""
+
+  git commit \
+      --signoff \
+      --message "\"Bump nightly toolchain to ${FOUND_VERSION}\""
 
   # This script runs from a pipeline step, thus we do not
   # have access to expeditor's open_pull_request bash
   # action helper function.
   install_hub
 
   echo "--- :github: Creating PR"
-  maybe_run hub pull-request --push --no-edit
+  hub pull-request --push --no-edit
```

Comment:
This won't work* until the github token provided to us by expeditor can push to non-chef organizations.  For now, perhaps we should just skip this step and revisit in the future?

*(May have changed since I last spoke to @tduffield in December)